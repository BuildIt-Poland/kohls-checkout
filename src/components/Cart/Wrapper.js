import styled from 'styled-components';

const Wrapper = styled.article`
  padding-bottom: 12rem;
`;

export default Wrapper;
