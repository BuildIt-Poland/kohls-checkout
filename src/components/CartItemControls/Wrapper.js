import styled from 'styled-components';

import { spacingMedium, demoYellow } from '../../styles/designTokens';

const Wrapper = styled.div`
  background: ${demoYellow};
  display: flex;
  justify-content: space-between;
  margin-top: ${spacingMedium};
`;

export default Wrapper;
