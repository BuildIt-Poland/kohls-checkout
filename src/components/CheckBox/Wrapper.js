import styled from 'styled-components';

import { spacingLarge, spacingSmall } from '../../styles/designTokens';

const Wrapper = styled.div`
  position: relative;
  display: block;
  padding-left: ${spacingLarge};
  margin: ${spacingSmall};
  cursor: pointer;
`;

export default Wrapper;
