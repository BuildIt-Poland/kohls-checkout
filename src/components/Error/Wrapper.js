import styled from 'styled-components';

import {
  colorRed,
  colorLightRed,
  colorBlack,
  fontMedium,
  fontWeightBold,
  spacingSmall,
  borderWidthThick
} from '../../styles/designTokens';

// TODO: Probably add generic InfoBox later @blurbyte
const Wrapper = styled.p`
  width: 100%;
  border: ${borderWidthThick} solid ${colorRed};
  color: ${colorBlack};
  background-color: ${colorLightRed};
  font-size: ${fontMedium};
  font-weight: ${fontWeightBold};
  padding: ${spacingSmall};
`;

export default Wrapper;
