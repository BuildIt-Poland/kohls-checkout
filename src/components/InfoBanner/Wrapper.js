import styled from 'styled-components';
import { colorTurquoise, colorWhite, fontMedium, spacingSmall } from '../../styles/designTokens';

const Wrapper = styled.section`
  display: flex;
  justify-content: center;
  padding: ${spacingSmall} 0;
  background-color: ${colorTurquoise};
  color: ${colorWhite};
  font-size: ${fontMedium};
  text-align: center;
`;

export default Wrapper;
