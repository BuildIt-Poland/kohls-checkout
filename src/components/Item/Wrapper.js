import styled from 'styled-components';

const Wrapper = styled.div`
  display: grid;
  grid-template-columns: 30% auto;
  align-items: center;
  margin: 3rem 1.3rem;
`;

export default Wrapper;
