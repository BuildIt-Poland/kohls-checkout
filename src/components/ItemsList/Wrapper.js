import styled from 'styled-components';

import { colorLightGrey, borderWidthThin, spacingLarge } from '../../styles/designTokens';

const Wrapper = styled.section`
  margin-top: ${spacingLarge};
  border-top: ${borderWidthThin} solid ${colorLightGrey};
  padding-top: ${spacingLarge};
`;

export default Wrapper;
