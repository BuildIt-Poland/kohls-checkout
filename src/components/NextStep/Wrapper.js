import styled from 'styled-components';

import {
  colorWhite,
  colorLightGrey,
  borderWidthThin,
  spacingSmall,
  zIndexStickyElement
} from '../../styles/designTokens';

const Wrapper = styled.div`
  bottom: 0;
  left: 0;
  padding: ${spacingSmall};
  position: fixed;
  width: 100%;
  background: ${colorWhite};
  border-top: ${borderWidthThin} solid ${colorLightGrey};
  z-index: ${zIndexStickyElement};
`;

export default Wrapper;
