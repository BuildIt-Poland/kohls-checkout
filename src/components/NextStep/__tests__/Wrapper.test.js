import React from 'react';
import { create } from 'react-test-renderer';

import Wrapper from '../Wrapper';

describe('COMPONENT - NextStep Wrapper', () => {
  it('render NextStep component', () => {
    const component = create(<Wrapper />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
