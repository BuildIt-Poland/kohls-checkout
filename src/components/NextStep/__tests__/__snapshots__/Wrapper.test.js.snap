// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - NextStep Wrapper render NextStep component 1`] = `
.c0 {
  bottom: 0;
  left: 0;
  padding: 1rem;
  position: fixed;
  width: 100%;
  background: #ffffff;
  border-top: 1px solid #b0b0b0;
}

<div
  className="c0"
/>
`;
